### ISSUE TITLE
Duplicate Odger's Teeth count towards Magical Power

### ISSUE THREAD

**Issue by MattTheCuber** [SkyCryptWebsite/SkyCrypt#1432]
**Describe the bug**
Accessories -> Magic Power uses incorrect accessories. It lists all of his accessories showing 0 command and 17 uncommon. But, in the Magic Power dropdown it calculates based off of 1 common and 18 uncommon. I am fairly certain this is due to the duplicate Odger's teeth accessories.

**Profile/Location**
[Deathstreeks](https://sky.shiiyu.moe/stats/DeathStreeks/Blueberry#Accessories)

**Additional Info**
He has duplicate Odger's teeth (Bronze, Silver, and Gold).

**Expected behavior**
Should say 1,115 Magic Power and use 0 common and 17 uncommon accessories.

**Screenshots**
![image](https://user-images.githubusercontent.com/32849887/170867606-a7b0c1f9-34bb-4d14-9c02-563da637e101.png)

**Comment by lochyj** [comment]
TLDR; duplicate Odger's teeth count towards magical power
... Also consider renaming this issue to something more descriptive of the issue such as "Duplicate Odger's teeth count towards magical power"

**Comment by MartinNemi03** [comment]
Adding to this: Seems like the magical power also counts for previous talismans in a given accessory family

**Comment by DuckySoLucky** [comment]
@metalcupcake5
